### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by EdvinasA**
Veikia Mix su sudėtim, atimtim, dalyba ir daugyba, neveikia su faktorial, bet atskirai factorial veikia.

### CODE REVIEW

File: tests/edvinas/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        return factorial;
+    }
+
+    public static void printArray (String [] array) {
+        for (int i = 0; i < array.length; i++) {
+            System.out.println(array[i]);
+        }
+    }
+
```
Comment (rimux9) [inline tests/edvinas/Calculator.java:64]: 👍

**Review by rimux9 — COMMENTED** [review]
good!

File: tests/edvinas/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+class Calculator{
+    public static void main(String[] args) {
+
+        System.out.println(calculator("5+5") + factorial(5));
+
+//        System.out.println(factorial(5));
+
+    }
+
+    public static int calculator(String input) {
```
Comment (Almantask) [inline tests/edvinas/Calculator.java:12]: Very hard to read, maybe you could split stuff by blank lines or maybe into functions?

Comment (EdvinasA) [reply tests/edvinas/Calculator.java:12]: Better ?

**Pull request closed without merging**